react/index.web.js

```diff
@@ -9,12 +9,27 @@ import DialInSummaryApp from './features/invite/components/dial-in-summary/web/D
 import PrejoinApp from './features/prejoin/components/web/PrejoinApp';
 
 const logger = getLogger('index.web');
-const OS = Platform.OS;
+
+// Add global loggers.
+window.addEventListener('error', ev => {
+    logger.error(
+        `UnhandledError: ${ev.message}`,
+        `Script: ${ev.filename}`,
+        `Line: ${ev.lineno}`,
+        `Column: ${ev.colno}`,
+        'StackTrace: ', ev.error?.stack);
+});
+
+window.addEventListener('unhandledrejection', ev => {
+    logger.error(
+        `UnhandledPromiseRejection: ${ev.reason}`,
+        'StackTrace: ', ev.reason?.stack);
```

Comment:
Usually a promise will be rejected with an Error, so let's log the stack, but it might not always be there.

Looging the promise seems pretty useless since it won't tell us anything so I'm skipping it.

Reply:
Agree about the promise!

I see that before we were trying to extract the message from the error. We don't need this?

Reply:
`reason` is the error object, or a string, or whatever the promise was rejected with, so we just serialize that.